aiohttp/client.py

```diff
@@ -526,7 +527,7 @@ async def _request(
                     assert conn.protocol is not None
                     conn.protocol.set_response_params(
                         timer=timer,
-                        skip_payload=method.upper() == "HEAD",
+                        skip_payload=method_must_be_empty_body(method.upper()),
```

Comment:
Probably a good idea to put the .upper() in the helper?

Reply:
adjusted in d1fb2c103a9be5683a93e3106e06247bca0b3c5e